.approvals

```diff
@@ -0,0 +1,3 @@
+spec/approvals/definitioncoercibletype/with_incoercible_nil_value/behaves_like_it_does_not_conform/has_correct_error_hash.approved.json spec/approvals/definitioncoercibletype/with_incoercible_nil_value/behaves_like_it_does_not_conform/has_correct_error_hash.received.json
```

Comment:
i guess this should not be committed. not 100% sure but i think the whole file could be removed and added to .gitignore?

Reply:
Removed it